src/shims/lock.h

```diff
@@ -95,7 +96,11 @@ typedef pid_t dispatch_lock_owner;
 #define DLOCK_OWNER_MASK			((dispatch_lock)FUTEX_TID_MASK)
 #define DLOCK_WAITERS_BIT			((dispatch_lock)FUTEX_WAITERS)
 #define DLOCK_FAILED_TRYLOCK_BIT	((dispatch_lock)FUTEX_OWNER_DIED)
-#define _dispatch_tid_self()		/* FIXME cached in TSD in the swift port */
+
+#include <unistd.h>
```

Comment:
please group the `#include`'s together after the `defined(__linux__)` and keep the order of the `#define _dispatch_tid_self()` to mirror the order of defines for OS X